packages/pillarbox-playlist/src/pillarbox-playlist.js

```diff
@@ -78,13 +97,20 @@ class PillarboxPlaylist extends Plugin {
    */
   constructor(player, options) {
```

Comment:
- the `options` jsdoc should describe permitted/expected properties
- we should maybe set `options` to `{}` to prevent some issues
- ```const player = pillarbox('player'); player.pillarboxPlaylist();``` seems to break everything. I checked the tests and don't understand why they're not failling.

Reply:
Good catch! I've added the jsdoc, the 2 issues were already solved in this PR: #25 it's waiting for review.